### DISCUSSION TITLE
Swapping info from "Details" category of the map and info on the map's thumbnail

### DISCUSSION THREAD

**Discussion by Highmagik** [ppy/osu#21626] [Ideas] [answered]
I like to look at a leadearbord of a beatmap while scrolling through maps, but I also want to see map's stats like AR, OD, HP and CS. Having those parameters hidden behind "Details" category feels inconvenient. I need to constantly switch between seeing rankings or stats. And there is a place for these params on the map's thumbnail.
<details>
   <summary>This is what I mean</summary>

![image](https://user-images.githubusercontent.com/71317439/207211933-f859d2f6-8fdf-4a0f-9a9a-1a82bf4f97c2.png)
</details> Instead of object countings which aren't that useful compared to AR and such but for some reason always in sight I suggest placing AR, OD, HP and CS on the thumbnail.

**Comment by Highmagik** [comment]
Basically, I want it to be done like it's done in stable (hope it's not a grave sin here)

**Comment by smoogipoo** [comment, marked as answer]
This will be improved with the song select redesign: https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1

**Comment by ikkue** [comment]
Temporary solution is to use the `BeatmapArtributeText` added in the last update and manually skin the song select menu.
